queries.js

```diff
@@ -76,11 +85,17 @@ module.exports = (self, query) => {
             .upcoming(null)
             .toDistinct('startDate');
 
-          const years = [{ value: null, label: 'All' }];
+          const years = [ {
+            value: null,
+            label: 'All'
+          } ];
           for (const eachdate of alldates) {
```

Comment:
Not a big deal because they are local variables, but `eachDate` and `allDates` read better.